gcloud/monitoring/timeseries.py

```diff
@@ -123,6 +140,32 @@ def __repr__(self):
         )
 
 
+def _get_typed_value_from_value(value):
+    """Infers the typed value for a point from the Python type.
+
+    Typed values contain a string representing the type of the
+    value being written, and the value itself. They are used when writing
+    points to time series. This method returns the appropriate string to
+    use when writing typed values based on the Python type of the value.
+
+    See: https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TypedValue
+
+   :type value: bool, int, float, str, or dict
+   :param value: value to infer the typed value of.
+
+   :rtype: str
+   :returns: The string that signifies the appropriate typed value.
+    """
+    typed_value_map = {
```

Comment:
Leaning against it due to a preference for keeping scope of variables as narrow as possible, however if you feel strongly I can still change.